mmdet/models/backbones/pvt.py

```diff
@@ -6,7 +6,8 @@
 import torch.nn as nn
 import torch.nn.functional as F
 from mmcv.cnn import (Conv2d, build_activation_layer, build_norm_layer,
-                      constant_init, normal_init, trunc_normal_init)
+                      constant_init, normal_init, trunc_normal_,
```

Comment:
`trunc_normal_` cannot be imported from `mmcv.cnn`, please import it from `mmcv.cnn.utils.weight_init ` or use `nn.init.trunc_normal_ ` instead.

Reply:
resolved.